src/extension.ts

```diff
@@ -34,7 +34,10 @@ const COMMON_ERRORLOG_PATTERNS = [
 
 // TODO: Refactor to use activeEditor
 export function activate(context: sourcegraph.ExtensionContext): void {
-    sourcegraph.workspace.openedTextDocuments.subscribe(textDocument => {
+    concat(
+        ...sourcegraph.workspace.textDocuments.map(doc => of(doc)),
+        from(sourcegraph.workspace.onDidOpenTextDocument)
```

Comment:
`onDidOpenTextDocument` is deprecated.

The right fix here would be to only use the `activeEditor` Observable you declare below, which is composed using `activeWindowChanges` and `activeViewComponentChanges`, which both emit their current value on susbcription, allowing you to never "miss" a document.

If you need the document content, you can access it using `CodeEditor.document`.

Additionally, you should make sure to pass all subscriptions to `context.subscriptions.add()`.